streams.json

```diff
@@ -38,10 +42,12 @@
           "repository_type": "ARCHIVE",
           "repository_url": "http://www.antlr2.org/download/antlr-2.7.7.tar.gz",
           "codebase": "",
-          "tag": "2.7.7",
+          "tag": "",
+          "group_id": "antlr",
+          "artifact_id": "antlr",
           "version": "2.7.7",
-          "gav": "antlr",
-          "comment": "Tag is just a guess"
+          "gav": "antlr:antlr:2.7.7",
+          "comment": "TAG not found. Tag is just a guess"
```

Comment:
Inconsistent with the tag removed further up.

Reply:
We can add the tag manually to the dictionary instead. Since the URL is an archive, we are not able to get a TAG from there